getting-started/templates/systemlink-secrets.yaml

```diff
@@ -162,7 +162,7 @@ userservices:
       - serviceName: "webserver"
         key: ""
         hash: "<SHA512 hash of key>"
-      - serviceName: "workorder"
```

Comment:
It's not great that a key that was only required for manual testing was added to this file. Should we include something in the release notes suggesting that this key be deleted from customer installs?

Reply:
I'll include something in the 2025-10 release notes.